bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -1341,7 +1349,10 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
 
         if (msg instanceof BookieProtocol.Response) {
             BookieProtocol.Response response = (BookieProtocol.Response) msg;
-            readV2Response(response);
+            if (!responses.offer(response)) {
```

Comment:
It will increase the latency of the latency-sensitive workloads.
At least this has to be behind a feature flag.

Reply:
Good point. I add a flag to enable this feature.